Makefile

```diff
@@ -51,11 +51,21 @@ EXT_JERRY_FLAGS ?=	-DENABLE_ALL_IN_ONE=ON \
 					-DFEATURE_PROFILE=$(ZJS_BASE)/jerry_feature.profile \
 					-DFEATURE_ERROR_MESSAGES=OFF \
 					-DJERRY_LIBM=OFF
-ifneq ($(DEV), ashell)
+# Settings for ashell builds
+ifneq (,$(filter $(MAKECMDGOALS),ide ashell))
+SNAPSHOT=off
```

Comment:
I don't think this is needed, right? It's already off. If it's on, you're about to give an error.